### ISSUE TITLE
Email RCPCH icon not loading

### ISSUE THREAD

**Issue by dc2007git** [rcpch/rcpch-audit-engine#694]
Just logged onto the live site and received my OTP, looks like the RCPCH icon isn't loading in the body

<img width="631" alt="image" src="https://github.com/rcpch/rcpch-audit-engine/assets/65614251/c0d0baa1-0156-43de-9a30-515526175dee">

**Comment by pacharanero** [comment]
Thanks @dc2007git for creating the Issue - yes I have noticed the same too. It comes from when I did some refactoring of the emails to improve consistency of style across the different emails that the platform sends.

If you inspect the original source of the OTP email you can see in the HTML version there is this section:

```html
<div class="centered" style="width: 50%; margin: auto;">
    <img alt="RCPCH Epilepsy12 Logo" src=':///static/images/epilepsy12-logo-1.3b08f7bf0eb0.png'>
  </div>
```

The image `src` is clearly malformed and I suspect it is because something is not being evaluated in the part of the template which supplies `{{ protocol }}` and `{{ domain }}` as below:

```html
# rcpch-audit-engine/templates/two_factor/email_token.html
  <div class="centered" style="width: 50%; margin: auto;">
    <img alt="RCPCH Epilepsy12 Logo" src='{{ protocol }}://{{ domain }}{% static "images/epilepsy12-logo-1.png" %}'>
  </div>
```

The exact same template code is being properly evaluated by Django in all the other email templates, I think this one is unusual as it is part of the 2FA module. It could be that `protocol` and `domain` are not being passed into this template, but are in the others.

Options to fix would be (in roughly order of preference):
1. find out why `protocol` and `domain` don't get evaluated and fix it so they do
2. hard code a link to that image somewhere eg `https://github.com/rcpch/rcpch-audit-engine/blob/development/static/images/epilepsy12-logo-1.png` which will always work, but should be considered a temporary fix
3. remove the image so it doesn't confuse users

**Comment by pacharanero** [comment]
This was closed in #708
